emcc.py

```diff
@@ -1099,6 +1099,8 @@ def check(input_file):
       if shared.Settings.LEGACY_VM_SUPPORT:
         # legacy vms don't have wasm
         assert not shared.Settings.WASM, 'LEGACY_VM_SUPPORT is only supported for asm.js, and not wasm. Build with -s WASM=0'
+        shared.Settings.POLYFILL_OLD_MATH_FUNCTIONS = 1
+        shared.Settings.SUPPORT_IOS_9 = 1
```

Comment:
The IOS option has a different name here than elsewhere, which is `WORKAROUND_IOS_9_RIGHT_SHIFT_BUG`